cogs/utils/dataIO.py

```diff
@@ -9,6 +9,15 @@ class InvalidFileIO(Exception):
 class DataIO():
     def __init__(self):
         self.logger = logging.getLogger("red")
+        try:
+            __import__("aiofiles")
+            import aiofiles
+            self.async_compatible = True
+        except ImportError:
+            self.logger.exception("Attempted to use aiofiles but "
```

Comment:
we have a `bot.pip_install` thing. since it's kinda important for it to be there, should we just use that instead of just logging?